torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -939,6 +940,31 @@ def test_new_subgroups_overlap_not_allowed(self):
                     ranks_per_subgroup_list=[[0], [1, 2], [1, 3]]
                 )
 
+        @require_backend({"nccl"})
```

Comment:
I think we can enable this test for both gloo and nccl?

Reply:
Yes, updated the decorator.